File: Homeworks/01. C# Fundamentals/03. Operators-and-Expressions/05. Third digit/Thirddigit.cs
using System;

class ThirdDigit
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        int numberTwo = number / 100;
        int numberThree = numberTwo % 10;
        if (numberThree == 7)
            {
            Console.WriteLine("true");
            }
        else
        {
            Console.WriteLine("false {0}", numberThree);
        }
    }
}

File: Homeworks/06. JS Fundamentals/04. Operators and Expressions/04. Third Digit/ThirdDigit.js
function thirdDigitIs7(number) {
    var thirdDigit = Math.floor(number / 100) % 10;
    if (thirdDigit == 7){
    	console.log(true)
    }
    else {
    	console.log("false " + thirdDigit)
    }
}